pkg/identityserver/store/application.go

```diff
@@ -80,8 +80,8 @@ var applicationColumnNames = map[string][]string{
 
 func (app Application) toPB(pb *ttnpb.Application, fieldMask *pbtypes.FieldMask) {
 	pb.Ids = &ttnpb.ApplicationIdentifiers{ApplicationId: app.ApplicationID}
-	pb.CreatedAt = cleanTime(app.CreatedAt)
-	pb.UpdatedAt = cleanTime(app.UpdatedAt)
+	pb.CreatedAt = cleanTimePtr(&app.CreatedAt)
```

Comment:
In case any reviewer wants to ask: shouldn't this become nullable at model level too ?
The answer is no, since they are controlled by the stores, and they cannot be modified via RPCs/store updates.

Reply:
Actually, it's because they are NOT NULL columns in the database.